src/desktop/components/search_bar/view.coffee

```diff
@@ -64,11 +73,38 @@ module.exports = class SearchBarView extends Backbone.View
 
   checkSubmission: (e) ->
     @hideSuggestions()
+
     return if !(e.which is 13) or @selected?
 
     unless @isEmpty()
       @trigger 'search:entered', encodeURIComponent(@$input.val())
 
+  maybeHideSpotlight: (e) ->
+    @hideSpotlight() unless e.which is 37 or e.which is 39
+
+  hideSpotlight: ->
+    @$spotlight.css('z-index', '-1')
+
+  displaySpotlightSearch: (results) ->
```

Comment:
this is:

```
IF 
  more than 4 chars have been typed into the search input
AND 
  query is an exact leading edge match of the top result
AND
  query is more than 30% of the length of the top result
```

then we do the spotlight search. Coming from https://github.com/artsy/gravity/pull/11715 (doesn't use any score).